lib/MusicBrainz/Server/Controller/User/Edits.pm

```diff
@@ -158,6 +158,28 @@ sub autoedits : Chained('/user/load') PathPart('edits/autoedits') RequireAuth Hi
     );
 }
 
+sub applied : Chained('/user/load') PathPart('edits/applied') RequireAuth HiddenOnSlaves {
+    my ($self, $c) = @_;
+    $self->_edits($c, sub {
+        return $c->model('Edit')->find({
+            editor => $c->stash->{user}->id,
+            status => $STATUS_APPLIED,
+        }, shift, shift);
+    });
+    $c->stash(
+        refine_url_args =>
+            { auto_edit_filter => '', order=> 'desc', negation=> 0,
+              combinator=>'and',
```

Comment:
I know this was copied from above but it'd be nice to make the spacing around `=>` consistent and put each property on its own line.